sdks/python/apache_beam/dataframe/transforms.py

```diff
@@ -16,13 +16,28 @@
 
 from __future__ import absolute_import
 
+import typing
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Mapping
+from typing import Tuple
+from typing import TypeVar
+from typing import Union
+
 import pandas as pd
 
 import apache_beam as beam
 from apache_beam import transforms
 from apache_beam.dataframe import expressions
 from apache_beam.dataframe import frames  # pylint: disable=unused-import
 
+if typing.TYPE_CHECKING:
```

Comment:
The prevailing style for `TYPE_CHECKING` is to import it as `from typing import TYPE_CHECKING`.  I think we should stay consistent.  If we want to change that, it's fine by me, but we can do that in another PR.

Reply:
+1 for consistency. Changed.